R/MeasureFairness.R

```diff
@@ -44,6 +45,7 @@ MeasureFairness = R6Class("MeasureFairness", inherit = Measure, cloneable = FALS
       super$initialize(
         id = paste0("fairness.", base_measure$id),
         range = c(-Inf, Inf),
+        properties = "reqiures_task",
```

Comment:
This properties do not pass in the measures somehow. eg:
```r
me = MeasureFairness$new("groupwise_abs_diff", base_measure)
me$properties
>>> character(0)
```

@mllg Could you please take a look? It took me about an hour to debug but I cannot find the reasons.

Reply:
This works for me, maybe you forgot to reload the package?

Reply:
@superp0tat0 check this works in the unit tests

Reply:
I see. Resolved the issue after I rebuild and install the whole package.